docs/UserGuide.md

```diff
@@ -751,13 +751,13 @@ Listed all meetings
 
 ## Editing an Existing Meeting
 
-Edits an existing meeting detail that is in the list.
+Edits an existing meeting in the meeting list. The existing meeting to be editted will have its parameters updated with the new values provided by the user.
 
 - Command: `edit-m`
-- Format: `edit-m <INDEX_NUMBER> b/BIDDER_ID p/PROPERTY_ID v/VENUE t/DATE`
+- Format: `edit-m <INDEX_NUMBER> b/BIDDER_ID p/PROPERTY_ID v/VENUE d/DATE s/STARTTIME e/ENDTIME`
 
 > - Edits the meeting at the specified `INDEX_NUMBER`, which refers to the index shown on the displayed meeting list. The index must be a positive integer 1, 2, 3...
-
+More than one parameter can be updated for the existing meeting to be editted.
```

Comment:
this refer to dianne property got a line that says smth like at least one optional parameter must be given

Reply:
Fixed